packages/web/tailwind.config.js

```diff
@@ -161,6 +161,8 @@ module.exports = {
         "linear-gradient(to bottom,rgba(0, 0, 0, 0),rgba(0, 0, 0, 0.8));",
       "gradient-dummy-notifications":
         "linear-gradient(0deg, #282750 0%, rgba(40, 39, 80, 0.00) 100%)",
+      "gradient-token-details-info":
```

Comment:
this is unused, i might have to remove it

Reply:
[da1cbbe](https://github.com/nabla-studio/osmosis-frontend/pull/22/commits/da1cbbea54e7a1137ed59235bc9239269c2e2773)